File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        mp = {}
        for i in range(len(nums)):
            want = target - nums[i]

            if want in mp:
                return [mp[want], i]
            
            mp[nums[i]] = i
        
        return [-1. -1]

File: Data Structures & Algorithms/remove-element/submission-1.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int k = 0;

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != val) {
                int tmp = nums[k];
                nums[k] = nums[i];
                nums[i] = tmp;
                k++;
            }
        }

        return k;
    }
}